google/ggrc-core
# GGRC-8235 Update Babel to use last 2 Chrome versions

# Issue description

Babel transforms the code from the modern js syntax to the oldest in order to the old versions of the browsers work correctly. Actually, the .babelrc contains the instructions that the code should be transformed to the 2 latest versions of any existing browser. However, our app is mostly focused on Chrome browser which supports any new features (babel is not required for them).

# Steps to test the changes

1. Run the app.
2. Verify that everything works properly. 

# Solution description

1. Updated Babel.
2. Resolved issues with cycle dependencies. 

# Sanity checklist




# PR Review checklist
